resources/qml/dialogs/ImagePackDeleteDialog.qml

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import Qt.labs.platform 1.1 as P
+import QtQuick
+import im.nheko
+
+P.MessageDialog {
+    id: deleteStickerPackRoot
+
+    property SingleImagePackModel imagePack
+    property ImagePackListModel packList
+
+    text: "This will delete the sticker pack: " + imagePack.packname
+    informativeText: "Are you sure you wish to delete the Sticker pack?"
+    modality: Qt.ApplicationModal
+    flags: Qt.Tool | Qt.WindowStaysOnTopHint | Qt.WindowCloseButtonHint | Qt.WindowTitleHint
+    buttons: P.MessageDialog.Yes | P.MessageDialog.No
+        
+    onAccepted: {
+        console.info("DELETE PACK: " + imagePack.packname);
```

Comment:
I'd suggest changing that to `"deleting image pack " + imagePack.packname` for maximum clarity. (It also would fit well with the current log style of everything lowercase :smile:)